config.ini

```diff
@@ -0,0 +1,9 @@
+[Default]
+PORT: 8081
+HOST: 127.0.0.1
```

Comment:
This can be removed as it's not actually used anywhere (was used in the sdx_decrypt_url, which is used by other repos)